src/main/java/spark/resource/AbstractResource.java

```diff
@@ -120,7 +120,7 @@ public File getFile() throws IOException {
     @Override
     public long contentLength() throws IOException {
         InputStream is = this.getInputStream();
-        Assert.state(is != null, "resource input stream must not be null");
+        Assert.isTrue(is == null, "resource input stream must not be null");
```

Comment:
The logic and the comment don't match.  If this.getInputStream() returns null, the equality will be true, and the assert will pass.  But the comment says input stream must NOT be null.